e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -132,9 +137,10 @@ describe("scenarios > search", () => {
     });
   });
 
-  describe("applying search filters", () => {
+  describeEE("applying search filters", () => {
```

Comment:
I don't think you should wrap everything inside `describeEE`. If you look at the current file on `master` https://github.com/metabase/metabase/blob/0f65cde6e714475995f126a0613730bc3694f92d/e2e/test/scenarios/onboarding/search/search.cy.spec.js

There are 11 `it()`s (1 of which is for a snowplow test)

After changing this to `describeEE`, only 4 tests are run on the opensource edition.
![image](https://github.com/metabase/metabase/assets/1937582/464fd335-c907-4ac0-8af6-895724181630)

When comparing this to the run on master which has 17 tests (not counting the snowplow one). This change reduce the coverage on the open source edition significantly.
![image](https://github.com/metabase/metabase/assets/1937582/29a05ea0-e7d0-4919-a8d6-9c96b36ea3eb)

I think you should extract the enterprise test to a new `describeEE` block and leave the rest of the tests the same.

Reply:
Cool, I moved the tests around to test per-filter, and make sure that the `Verified` filter is the only one that relies on `describeEE` since that's the only one that is conditional on that